gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,9 +1,6 @@
-#Fri Jun 01 19:51:59 EST 2018
+#Tue Oct 09 03:48:35 IST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-android.enableD8.desugaring=true
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
-# Android Studio Gradle wrapper 3.1.2 doesn't support configure on demand
-org.gradle.configureondemand=false
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```

Comment:
are changes with `gradle-wrapper.properties` necessary?

Reply:
Reverted the commit that made these changes. Although, these were made to build faster on my system. I understand these should not have been pushed. 
Apologies.
Thanks @huangyz0918